Makefile

```diff
@@ -22,18 +19,11 @@
 SRC_DIR = src/main/scala
 MODULE = Core
 
-# FPGA
+# Directories
 FPGA_DIR = fpga
-
 EMULATOR_DIR = emulator
 SCRIPTS_DIR = scripts
 
-# Mill build tool for compiling and running the Chisel generator.
-MILL_VERSION = 0.6.0
-MILL_BIN = ./mill
-$(MILL_BIN):
-	wget https://github.com/lihaoyi/mill/releases/download/$(MILL_VERSION)/$(MILL_VERSION) -O $(MILL_BIN) && chmod +x $(MILL_BIN)
-
```

Comment:
Do you mind if we keep this as I do personally prefer using mill over sbt?

Reply:
Sounds good. I will take care of this.